blotztask-api/Modules/Tasks/Queries/SubTasks/GetSubtasksByTaskId.cs

```
@@ -0,0 +1,54 @@
+using System.ComponentModel.DataAnnotations;
+using BlotzTask.Infrastructure.Data;
+using BlotzTask.Modules.BreakDown;
+using Microsoft.EntityFrameworkCore;
+
+namespace BlotzTask.Modules.Tasks.Queries.SubTasks;
+
+
+public class GetSubtasksByTaskIdQuery
+{
+    [Required]
+    public Guid UserId { get; init; }
+    [Required]
+    public int TaskId { get; init; }
+}
+
+public class SubTasksDto
+{
+    public int Id { get; init; }
+    public string Title { get; init; }
+    public bool IsDone { get; init; }
+    public int Order { get; init; }
+}
+
+public class GetSubtasksByTaskIdQueryHandler(BlotzTaskDbContext db, ILogger<GetSubtasksByTaskIdQueryHandler> logger)
+{
+    public async Task<IEnumerable<SubTasksDto>> Handle(GetSubtasksByTaskIdQuery query, CancellationToken ct)
+    {
+        var task = await db.TaskItems
+            .FirstOrDefaultAsync(t => t.Id == query.TaskId && t.UserId == query.UserId, ct);
+        if (task is null)
```

this should throw error if task id not found